### PR TITLE
feat(docs) - documentation of exception types - OperationFailed, OperationRejected, BadRequest

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: wiki/Manual.md [hunk 1]
```diff
@@ -6339,43 +6337,48 @@ Here is an outline of exception inheritance hierarchy: https://github.com/ccxt/c
...
+From `BaseError` derives two different families of errors: `OperationFailed` and `ExchangeError` (they also have their specific sub-types, as explained below).
 
-They also have more specific sub-type exceptions, as explained below.
+### OperationFailed
+<a name="NetworkError" id="NetworkError"></a>
 
-### NetworkError
+An `OperationFailed` happens when user sends **correctly constructed & valid request** to exchange, but an non-deterministic problem occurred. In suchcases, you might consider retrying after some time.
+- maintenance ongoing
+- internet/network connectivitiy issues
```
Comment (ttodua) [inline wiki/Manual.md:6367]:
```suggestion
- rate limits exceeded
- internet/network connectivitiy issues
```

shouldnt we include RL note?

Comment (carlosmiei) [reply wiki/Manual.md:6367]: I don't think so we have a specific exception for that, I think it might confuse the users

File: wiki/Manual.md [hunk 2]
```diff
@@ -6339,43 +6337,48 @@ Here is an outline of exception inheritance hierarchy: https://github.com/ccxt/c
...
-- `NetworkError`
-- `ExchangeError`
+From `BaseError` derives two different families of errors: `OperationFailed` and `ExchangeError` (they also have their specific sub-types, as explained below).
 
-They also have more specific sub-type exceptions, as explained below.
+### OperationFailed
+<a name="NetworkError" id="NetworkError"></a>
 
-### NetworkError
+An `OperationFailed` happens when user sends **correctly constructed & valid request** to exchange, but an non-deterministic problem occurred. In suchcases, you might consider retrying after some time.
```
Comment (ttodua) [inline wiki/Manual.md:6365]:
```suggestion
An `OperationFailed` happens when user sends **correctly constructed & valid request** to exchange, but an non-deterministic problem occurred. In such cases, you might consider retrying after some time.
```

File: wiki/Manual.md (hunk 1 above)
Comment (ttodua) [reply wiki/Manual.md:6367]: `RateLimitExceeded` is that.. ?

Comment (ttodua) [reply wiki/Manual.md:6367]: so we should list them under "temporary fails". it's the place where it belongs. maybe in opposite, missing that from the list might confuse users

File: wiki/Manual.md (hunk 2 above)
Comment (carlosmiei) [reply wiki/Manual.md]: yeah yeah noticed it also, already fixed

File: wiki/Manual.md (hunk 1 above)
Comment (carlosmiei) [reply wiki/Manual.md:6367]: The thing is, we have a more specific exception to map the RL error (NetworkError -> DDOSPRotection -> RlExcedeed), we're already explaining the broader idea behind `OperatioNFailed` I think that is enough, the simpler the better

Comment (ttodua) [reply wiki/Manual.md:6367]: the problem is that we dont have correct mapping (RL shoud not be under DDOS) so missing "rate limit issues" might confuse users into thinking as if "rate limits issues" is not listed under OpFailed. but sure, as you consider 👍

Comment (ttodua) [reply wiki/Manual.md:6369]: now, as we moved our RL and made it sibling, we can place in that list. not?

**Pull request merged**